src/api.ts

```diff
@@ -93,5 +96,10 @@ export function makeConsentManagerAPI({
     },
   };
 
-  return api;
+  const consentManagerAPI: ConsentManagerAPI = Object.assign(
+    eventTarget,
+    consentManagerMethods,
+  );
```

Comment:
To keep our API branding, specify `[Symbol.toStringTag]: 'Transcend'` in `consentManagerMethods`.